sktime/transformers/resizing.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+import pandas as pd
+from scipy import interpolate
+
+from sktime.transformers.base import BaseTransformer
+from sktime.utils.validation.supervised import validate_X
+
+
+class TSResizeTransform(BaseTransformer):
+    """Transformer that rescales series for another number of points. 
+        For each cell in datadrame transformer fits scipy linear interp1d 
+        and samples user defined number of points. Points are generated 
+        by numpy.linspace. After transformation each cell will be a numpy.array
+        of defined size.
+    """
+

```

Comment:
please add `_required_parameter = ["length"]` as a class attribute
```suggestion
         _required_parameter = ["length"]
```